src/components/flat-table/sort/sort.component.tsx

```diff
@@ -11,10 +13,20 @@ export interface SortProps {
   onClick?: () => void;
   /** Sets the content of `FlatTableSortHeader` */
   children?: React.ReactNode;
+  /**
+   * Creates an accessible name for `FlatTableSortHeader`, it is recommended that the
+   * text content of the `FlatTableSortHeader` as well as the current `sortType` are included
+   * in this accessible name
```

Comment:
**suggestion**: it might be helpful to add an example to our docs demonstrating how to use the `accessibleName` prop. What do you think about using a Storybook story to describe how it should be used instead?